config/locales/en.yml

```diff
@@ -27,6 +27,11 @@ en:
       'no': 'No'
       unknown: *dont_know
 
+    GENDER: &GENDER
```

Comment:
No need for this, the gender locales are already part of the `PERSONAL_DETAILS_FIELDS` dictionary.